Automattic/woocommerce-payments
# Initial type declarations

- Introduce type definitions for some of the common components fo WCPay
 - Provide guidelines on adding new type definitions.
 
 ## Testing instructions
 
  - All JS tests should pass
  - Make sure to rebuild client with `npm run build:client` and take a quick tour of affected pages. Pages should load with content and do not introduce new JS errors.
